src/sentry/api/endpoints/group_notes.py

```diff
@@ -74,11 +75,20 @@ def post(self, request: Request, group) -> Response:
             group=group, user_ids=mentioned_users["users"], reason=GroupSubscriptionReason.mentioned
         )
 
-        GroupSubscription.objects.bulk_subscribe(
-            group=group,
-            user_ids=mentioned_users["team_users"],
-            reason=GroupSubscriptionReason.team_mentioned,
-        )
+        if features.has(
```

Comment:
similar problem to the previous feature flag issue, we can still just drop the third argument since we won't use it.